python-frontend/src/main/java/org/sonar/python/semantic/v2/converter/PythonTypeToDescriptorConverter.java

```diff
@@ -42,7 +44,7 @@ public class PythonTypeToDescriptorConverter {
 
   public Descriptor convert(String moduleFqn, SymbolV2 symbol, Set<PythonType> types) {
     var candidates = types.stream()
-      .map(type -> convert(moduleFqn, moduleFqn, symbol.name(), type))
+      .map(type -> convert(moduleFqn, moduleFqn, symbol.name(), type, symbol))
```

Comment:
_personal preference_ I find it a bit weird to have `symbol` and `symbolName` as parameter of convert. A symbol will contain the symbolName so I would guess that we don't need to pass the `symbolName` parameter. 
I would be in favor of changing the signature of `convert` to 
``` java
private static Descriptor convert(String moduleFqn, String parentFqn, String symbolName, List<UsageV2> symbolUsages, PythonType type)
```
Then it will also simplify the logic since you won't have to check for null at line 66. WDYT?

Reply:
I actually started with this approach. However, class members don't have symbols and because of that, I was losing access to the necessary `symbolName` (for which `Member#name` is used instead on line 107).

I ended up duplicating the method for top-level types that have symbols and members, and it became a bit of a mess. I decided to just add the parameter instead.

Reply:
I think I agree with you but I was not explicit enough in my proposition: 
The idea was not to pass only `symbol` as a parameter and not passing `symbolName`. The idea was to replace the `symbol` parameter by `symbolUsages` keeping the `symbolName` parameter. I think this way it shouldn't be a problem even for members.

Reply:
Oh, sorry, I read the suggested code a bit too quickly.
Indeed, it makes a lot of sense, I made the change!